src/channel.rs

```diff
@@ -442,6 +442,23 @@ impl ChannelBuilder {
         let channel =
             unsafe { grpc_sys::grpc_insecure_channel_create(addr_ptr, args.args, ptr::null_mut()) };
 
+        self.connect_from_raw_channel(channel)
+    }
+
+    /// Build an insecure [`Channel`] taking over an established connection from
+    /// a file descriptor.
+    pub fn connect_from_fd(mut self, target: &str, fd: ::std::os::raw::c_int) -> Channel {
```

Comment:
It should be `unsafe` and unix only.

Reply:
Should explain what's target and who is responsible to open and close fd.

Reply:
thanks -- will amend doc, but how do I mark it "posix-only"?

Reply:
`#[cfg(unix)]`

Reply:
thanks, added as attribute